src/allmydata/storage/http_server.py

```diff
@@ -0,0 +1,94 @@
+"""
+HTTP server for storage.
+"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from future.utils import PY2
+
+if PY2:
+    # fmt: off
+    from future.builtins import filter, map, zip, ascii, chr, hex, input, next, oct, open, pow, round, super, bytes, dict, list, object, range, str, max, min  # noqa: F401
+    # fmt: on
+
+from functools import wraps
+
+from klein import Klein
+from twisted.web import http
+
+# TODO Make sure to use pure Python versions?
+from cbor2 import dumps
```

Comment:
I don't think you have to do anything to get pure-python version (well, except "run on PyPy")

Reply:
No, you do, it uses C version by default.